src/health/REFERENCE.md

```diff
@@ -1,58 +1,174 @@
-[infrastructure](/docs/netdata-cloud/organize-your-infrastructure-invite-your-team.md), you must configure each node with the same health configuration
-files.
+## Table of Contents
+- [Quick Start Guide](#quick-start-guide) <!-- SECTION_TYPE: tutorial -->
+- [Common Tasks](#common-tasks) <!-- SECTION_TYPE: workflow -->
+- [How-To Guides](#how-to-guides) <!-- SECTION_TYPE: procedures -->
+- [Alert Configuration Reference](#alert-configuration-reference) <!-- SECTION_TYPE: reference -->
+- [Expressions and Variables](#expressions-and-variables) <!-- SECTION_TYPE: concepts -->
+- [Alert Examples](#alert-examples) <!-- SECTION_TYPE: examples -->
+- [Troubleshooting](#troubleshooting) <!-- SECTION_TYPE: problem_solving -->
 
-## Reload health configuration
+## Quick Start Guide
 
-You don’t need to restart the Netdata Agent between changes to health configuration files, such as specific health entities. Instead, you can use `netdatacli` and the `reload-health` option to prevent gaps in metrics collection.
+:::tip 
 
+**What You'll Learn**
+
+In 5 minutes, you'll know how to edit existing alerts, create new ones, and reload your configuration without downtime.
+
+:::
+
+### Get Started in 3 Steps
+
+**Step 1: Find Your Config Directory**
+Navigate to your [Netdata config directory](/docs/netdata-agent/configuration/README.md)
+
+**Step 2: Edit an Alert**
+```bash
+sudo ./edit-config health.d/cpu.conf
+```
+
+**Step 3: Apply Changes**
 ```bash
 sudo netdatacli reload-health
 ```
 
-If `netdatacli` doesn't work on your system, send a `SIGUSR2` signal to the daemon, which reloads health configuration without restarting the entire process.
+:::note 
+
+**Key Concept**
+
+You can highly configure Netdata's health watchdog with support for dynamic thresholds, hysteresis, alert templates, and more. You can customize any existing alerts based on your infrastructure's topology or specific monitoring needs, or create entirely new entities.
+
+You can use health alerts with any of Netdata's [collectors](/src/collectors/README.md) (see the [supported collector list](/src/collectors/COLLECTORS.md)) to monitor your systems, containers, and applications in real time.
+
+While you can view active alerts on both the local dashboard and Netdata Cloud, you configure all health alerts _per node_ via individual Netdata Agents. If you want to deploy a new alert across your [infrastructure](/docs/netdata-cloud/organize-your-infrastructure-invite-your-team.md), you must configure each node with the same health configuration files.
+
+:::
+
+**Next Steps:** Jump to [Common Tasks](#common-tasks) for specific workflows, or continue reading for comprehensive guidance.
+
+
+## Common Tasks
+
+:::tip 
+
+**What You'll Learn**
+
+Step-by-step workflows for the most frequent alert configuration tasks.
+
+:::
+
+### Task 1: Modify Alert Thresholds
+
+**Why This Matters:** Default thresholds may not fit your specific environment or requirements.
+
+**Quick Example:**
+```yaml
```

Comment:
It is not YAML, it is Netdata's custom format. Use

````
```
abc
```
````

or

````
```text
abc
```
````

Reply:
easy